pkg/config/config.go

```diff
@@ -200,8 +200,9 @@ func init() {
 	Datadog.SetDefault("kubelet_client_key", "")
 
 	Datadog.SetDefault("kubernetes_collect_metadata_tags", true)
-	Datadog.SetDefault("kubernetes_metadata_tag_update_freq", 60*5) // 5 min
+	Datadog.SetDefault("kubernetes_metadata_tag_update_freq", 60) // Polling frequency of the Agent to the DCA: 1 min
 	BindEnvAndSetDefault("kubernetes_apiserver_client_timeout", 10)
+	BindEnvAndSetDefault("apiserver_poll_freq", 30)              // Polling frequency of the DCA to the API Server: 30 seconds
```

Comment:
I think `apiserver_poll_freq` needs the `kubernetes_` prefix?

Comments should say:

`Polling frequency of the Agent to the DCA (or apiserver) in seconds`

`Polling frequency of the DCA (or leader) to the apiserver in seconds`